src/fsharp/FSharp.Core/option.fsi

```diff
@@ -52,6 +81,15 @@ module Option =
     /// <param name="ifNone">The value to use if <paramref name="option"/> is <c>None</c>.</param>
     /// <param name="option">The input option.</param>
     ///
+    /// <example>
+    /// <code>
+    ///     (None, None) ||> Option.orElse // evaluates to None
```

Comment:
Note: this runs into value restriction, but in context I think (None, None) is clearer than ((None: int option), None).

Reply:
Agreed